components/src/dynamo/common/utils/video_utils.py

```
@@ -194,51 +244,43 @@ def encode_to_video_bytes(
-                    writer.append_data(frame)
-            finally:
-                writer.close()
-
-        video_bytes = buffer.getvalue()
-        logger.info(f"Encoded video to {len(video_bytes)} bytes")
-        return video_bytes
-
-    except Exception as e:
-        logger.error(f"Failed to encode video to bytes: {e}")
-        raise RuntimeError(f"Video encoding to bytes failed: {e}") from e
+            subprocess.run(cmd + [tmp.name], input=yuv, check=True, capture_output=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)